packages/core/getUserAvailability.ts

```diff
@@ -187,27 +187,67 @@ export async function getUserAvailability(
     source: query.withSource ? a.source : undefined,
   }));
 
-  const bookings = busyTimes.filter((busyTime) => busyTime.source?.startsWith(`eventType-${eventType?.id}`));
```

Comment:
@CarinaWolli bookings are derived from busyTimes, which does not include the whole month's booking and also use afterEventBuffer and beforeEventBuffer.